index.html

```diff
@@ -26,5 +26,139 @@ <h3 class="profession">I’m a Software Developer</h3>
         </ul>
     </section>
+    <section class="project-section">
+        <div class="my-work-title">
+            <h1>My Recent Works</h1>
+        </div>
+        <div id="line"></div>
+        <div id="works-section">
+            <div class="project1">
+                <h1 class="project1-title">Multi-Post-Stories</h1>
+                <p class="project1-description"> A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard dummy text
+                    ever since the 1500s, when an unknown printer took a standard
+                    dummy text.</p>
+                    <ul class="project1-tech">
+                        <li><button>css</button></li>
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project1-button">See Project</button>
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
```

Comment:
- [ ] please, kindly add the hover effect to your `see project` and `get my resume` buttons.On hover, the button color should change lightly to #FFF5E1 . This help the user to know he is on the button and thus making your portfolio user-friendly. Please fix that. see example below

![Screenshot (750)](https://user-images.githubusercontent.com/65298048/212431773-80061d34-2e23-49c9-9721-4aa3460386d2.png)